Tests/LibUnicode/TestUnicodeCharacterTypes.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include <AK/StringView.h>
 #include <LibUnicode/CharacterTypes.h>
+#include <LibUnicode/UnicodeData.h>
```

Comment:
We can't rely on this header existing. If someone turns the `ENABLE_UNICODE_DATABASE_DOWNLOAD` CMake option off, this fill will then not compile.

Most of the tests here work by doing an enum-from-string lookup, e.g.:
https://github.com/SerenityOS/serenity/blob/456211932fa27b25c62339e4fe79f4d0fa3bfd16/Tests/LibUnicode/TestUnicodeCharacterTypes.cpp#L609

Which is admittedly super annoying. If you'd prefer to avoid that, then this include and the test added below can go behind an `#if` guard:
```c++
#if ENABLE_UNICODE_DATA
#    include <LibUnicode/UnicodeData.h>
#endif
```

Reply:
Thanks! I went the "from_string()" route, it wasn't too bad.